front/src/common/Map/Layers/OSM.tsx

```diff
@@ -7,45 +7,49 @@ import mapStyleDarkJson from 'assets/mapstyles/OSMDarkStyle.json';
 import mapStyleBluePrintJson from 'assets/mapstyles/OSMBluePrintStyle.json';
 import { OSM_URL } from 'common/Map/const';
 
-import OrderedLayer from 'common/Map/Layers/OrderedLayer';
+import OrderedLayer, { OrderedLayerProps } from 'common/Map/Layers/OrderedLayer';
 
 interface OSMProps {
   mapStyle: string;
   layerOrder?: number;
 }
 
-function OSM(props: OSMProps) {
-  const { mapStyle, layerOrder } = props;
-
-  function getMapStyle(): LayerProps[] {
-    switch (mapStyle) {
-      case 'empty':
-        return [] as LayerProps[];
-      case 'dark':
-        return mapStyleDarkJson as LayerProps[];
-      case 'blueprint':
-        return mapStyleBluePrintJson as LayerProps[];
-      default:
-        return mapStyleJson as LayerProps[];
-    }
+export function getMapStyle(mapStyle: string): LayerProps[] {
```

Comment:
This function is not used outside this file
```suggestion
function getMapStyle(mapStyle: string): LayerProps[] {
```